lib/primitives/array-element.js

```diff
@@ -92,6 +92,15 @@ module.exports = function(BaseElement, registry) {
       return undefined;
     },
 
+    /*
+     * It reduces through a given array and calls get on each item.
+     */
+    getIn: function(paths) {
```

Comment:
This makes for a pretty complicated interface, IMO. You're OK with adding in this kind of magic? 

If we decide to do this, I think I will leave `getIn` and just delegate this functionality to it in `get`.